### PR TITLE
[Doc][Sqlserver]Add notice into Sqlserver cdc doc

### PR DESCRIPTION
**Pull request by zhangchengming601**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/connector-v2/sink/SQLServer.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+# SQL Server
+
+> JDBC SQL Server Sink Connector
+
+## Support those engines
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/SQLServer.md:5]: Please remove Support engines, connector-v2 always support all engine.

File: docs/en/connector-v2/sink/SQLServer.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+> Spark<br/>
+> Flink<br/>
+> Seatunnel Zeta<br/>
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/SQLServer.md]: Sink always support batch and stream

File: docs/en/connector-v2/sink/SQLServer.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+> Use `Xa transactions` to ensure `exactly-once`. So only support `exactly-once` for the database which is
+> support `Xa transactions`. You can set `is_exactly_once=true` to enable it.
+
+## Description
+
+Write data through jdbc. Support Batch mode and Streaming mode, support concurrent writing, support exactly-once
+semantics (using XA transaction guarantee).
+
+## Supported DataSource list
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/SQLServer.md]: It's weird, SQLServer sink only support SQLServer, why use `Supported DataSource list`?

File: docs/en/connector-v2/sink/SQLServer.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+| datasource | supported versions                                       | driver                                       | url                             | maven                                                        |
+| ---------- | -------------------------------------------------------- | -------------------------------------------- | ------------------------------- | ------------------------------------------------------------ |
+| SQL Server | Different dependency version has different driver class. | com.microsoft.sqlserver.jdbc.SQLServerDriver | jdbc:sqlserver://localhost:1433 | [Download](https://mvnrepository.com/artifact/com.microsoft.sqlserver/mssql-jdbc) |
+
+## Database dependency
+
+> Please download the support list corresponding to 'Maven' and copy it to the '$SEATNUNNEL_HOME/plugins/jdbc/lib/' working directory<br/>
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/SQLServer.md:32]: Same as above

File: docs/en/connector-v2/sink/SQLServer.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+## Task Example
+
+### simple:
+
+> This example defines a SeaTunnel synchronization task that automatically generates data through FakeSource and sends it to JDBC Sink. FakeSource generates a total of 16 rows of data (row.num=16), with each row having two fields, name (string type) and age (int type). The final target table is test_table will also be 16 rows of data in the table. Before run this job, you need create database test and table test_table in your SQL Server. And if you have not yet installed and deployed SeaTunnel, you need to follow the instructions in [Install SeaTunnel](../../start-v2/locally/deployment.md) to install and deploy SeaTunnel. And then follow the instructions in [Quick Start With SeaTunnel Engine](../../start-v2/locally/quick-start-seatunnel-engine.md) to run this job.
+
+```
+# Defining the runtime environment
+env {
+  # You can set flink configuration here
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/SQLServer.md:82]: Remove unnecessary comment.

File: docs/en/connector-v2/sink/SQLServer.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+  execution.parallelism = 1
+  job.mode = "BATCH"
+}
+
+source {
+  # This is a example source plugin **only for test and demonstrate the feature source plugin**
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/SQLServer.md:88]: Same as above

File: docs/en/connector-v2/sink/redshift.md [hunk 7]
```diff
@@ -0,0 +1,159 @@
+# Redshift
+
+> JDBC Redshift sink Connector
+
+## Support those engines
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/redshift.md:5]: Same as above

**Comment by Hisoka-X** [comment]
Each doc have same problems, please fix it. Thansk.

**Comment by zhilinli123** [comment]
Please close duplicate pull requests @zhangchengming601 
I added some review comments in this community boss help to open up :https://github.com/apache/incubator-seatunnel/pull/4715

File: docs/en/connector-v2/sink/SQLServer.md (hunk 1 above)
Comment (zhangchengming601) [reply docs/en/connector-v2/sink/SQLServer.md:5]: I wrote it according to document standards

File: docs/en/connector-v2/sink/SQLServer.md [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+
+## Support those engines
+
+> Spark<br/>
+> Flink<br/>
+> Seatunnel Zeta<br/>
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/SQLServer.md]:
```suggestion

```

File: docs/en/connector-v2/sink/SQLServer.md [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+
+> Use `Xa transactions` to ensure `exactly-once`. So only support `exactly-once` for the database which is
+> support `Xa transactions`. You can set `is_exactly_once=true` to enable it.
+
+## Description
+
+Write data through jdbc. Support Batch mode and Streaming mode, support concurrent writing, support exactly-once
+semantics (using XA transaction guarantee).
+
+## Supported DataSource list
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/SQLServer.md]:
```suggestion
## Supported DataSource Info
```

File: docs/en/connector-v2/sink/redshift.md [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+
+> Spark<br/>
+> Flink<br/>
+> Seatunnel Zeta<br/>
+
+## Key features
+
+- [ ] [batch](../../concept/connector-v2-features.md)
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/redshift.md]:
```suggestion

```

File: docs/en/connector-v2/sink/redshift.md [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+- [x] [exactly-once](../../concept/connector-v2-features.md)
+- [x] [cdc](../../concept/connector-v2-features.md)
+
+> Use `Xa transactions` to ensure `exactly-once`. So only support `exactly-once` for the database which is support `Xa transactions`. You can set `is_exactly_once=true` to enable it.
+
+## Description
+
+Write data through jdbc.
+
+## Supported DataSource list
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/redshift.md]:
```suggestion
## Supported DataSource Info
```

File: docs/en/connector-v2/source/SQLServer.md [hunk 12]
```diff
@@ -0,0 +1,154 @@
...
+supports query SQL and can achieve projection effect.
+
+- [x] [parallelism](../../concept/connector-v2-features.md)
+- [x] [support user-defined split](../../concept/connector-v2-features.md)
+
+## Description
+
+Read external data source data through JDBC.
+
+## Supported DataSource list
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/SQLServer.md]:
```suggestion
## Supported DataSource Info
```

File: docs/en/connector-v2/source/redshift.md [hunk 13]
```diff
@@ -0,0 +1,146 @@
...
+## Support those engines
+
+> Spark<br/>
+> Flink<br/>
+> Seatunnel Zeta<br/>
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
+- [x] [exactly-once](../../concept/connector-v2-features.md)
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/redshift.md:15]:
```suggestion
- [ ] [stream](../../concept/connector-v2-features.md)
- [x] [exactly-once](../../concept/connector-v2-features.md)
```

File: docs/en/connector-v2/source/redshift.md [hunk 14]
```diff
@@ -0,0 +1,146 @@
...
+- [x] [parallelism](../../concept/connector-v2-features.md)
+- [x] [support user-defined split](../../concept/connector-v2-features.md)
+
+> supports query SQL and can achieve projection effect.
+
+## Description
+
+Read external data source data through JDBC.
+
+## Supported DataSource list
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/redshift.md]:
```suggestion
## Supported DataSource Info
```

File: docs/en/connector-v2/source/SQLServer.md [hunk 15]
```diff
@@ -0,0 +1,154 @@
+# SQL Server
+
+> JDBC SQL Server Source Connector
+
+## Support those engines
+
+> Spark <br>
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/SQLServer.md]:
```suggestion
> Spark <br/>
```

File: docs/en/connector-v2/source/SQLServer.md [hunk 16]
```diff
@@ -0,0 +1,154 @@
...
+> Flink <br>
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/SQLServer.md]:
```suggestion
> Flink <br/>
```

File: docs/en/connector-v2/source/SQLServer.md [hunk 17]
```diff
@@ -0,0 +1,154 @@
...
+> Seatunnel Zeta <br>
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/SQLServer.md]:
```suggestion
> Seatunnel Zeta <br/>
```

File: docs/en/connector-v2/source/SqlServer-CDC.md [hunk 18]
```diff
@@ -167,6 +167,35 @@ source {
 
 Source plugin common parameters, please refer to [Source Common Options](common-options.md) for details.
 
+## notice
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/SqlServer-CDC.md:170]: I think update this doc to the new format is better.

**Comment by TyrantLucifer** [comment]
fix ci error

image

File: docs/en/connector-v2/sink/SQLServer.md [hunk 19]
```diff
@@ -0,0 +1,159 @@
...
+| CHAR<br />NCHAR<br />VARCHAR<br />NTEXT<br />NVARCHAR<br />TEXT | STRING                                                                                                                                                    |
+| DATE                                                            | LOCAL_DATE                                                                                                                                                |
+| TIME                                                            | LOCAL_TIME                                                                                                                                                |
+| DATETIME<br />DATETIME2<br />SMALLDATETIME<br />DATETIMEOFFSET  | LOCAL_DATE_TIME                                                                                                                                           |
+| TIMESTAMP<br />BINARY<br />VARBINARY<br />IMAGE<br />UNKNOWN    | Not supported yet                                                                                                                                         |
+
+## Options
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/SQLServer.md:52]: Sink Options

File: docs/en/connector-v2/sink/SQLServer.md [hunk 20]
```diff
@@ -0,0 +1,159 @@
...
+
+|             name             |  type  | required |     default     |                                                                                                                            Description                                                                                                                            |
+|------------------------------|--------|----------|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/SQLServer.md:55]: |             Name             |  Type  | Required |     Default     | Description|

File: docs/en/connector-v2/sink/SQLServer.md [hunk 21]
```diff
@@ -0,0 +1,159 @@
...
+source {
+  # This is a example source plugin **only for test and demonstrate the feature source plugin**
+  FakeSource {
+    parallelism = 1
+    result_table_name = "fake"
+    row.num = 16
+    schema = {
+      fields {
+        name = "string"
+        age = "int"
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/SQLServer.md:96]: Add SqlSerber so type testing is sure to work

File: docs/en/connector-v2/sink/SQLServer.md [hunk 22]
```diff
@@ -0,0 +1,159 @@
...
+
+```
+jdbc {
+    url = "jdbc:sqlserver://localhost:1433;databaseName=testdb"
+    driver = "com.microsoft.sqlserver.jdbc.SQLServerDriver"
+
+    max_retries = 0
+    user = "root"
+    password = "123456"
+    query = "insert into test_table(name,age) values(?,?)"
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/SQLServer.md:134]: ditto

File: docs/en/connector-v2/sink/redshift.md [hunk 23]
```diff
@@ -0,0 +1,159 @@
...
+source {
+  # This is a example source plugin **only for test and demonstrate the feature source plugin**
+  FakeSource {
+    parallelism = 1
+    result_table_name = "fake"
+    row.num = 16
+    schema = {
+      fields {
+        name = "string"
+        age = "int"
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/redshift.md:94]: ditto

File: docs/en/connector-v2/sink/redshift.md [hunk 24]
```diff
@@ -0,0 +1,159 @@
...
+| GEOMETRY                                                                                                          | Not supported yet                                                                                                                                   |
+
+## Options
+
+|             name             |  type  | required |     default     |                                                                                                                            description                                                                                                                            |
+|------------------------------|--------|----------|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/redshift.md:53]: ditto

File: docs/en/connector-v2/sink/redshift.md [hunk 25]
```diff
@@ -0,0 +1,159 @@
...
+| DATE                                                                                                              | LOCALDATE                                                                                                                                           |
+| TIME<br />TIME_WITH_TIME_ZONE<br />TIMETZ                                                                         | LOCALTIME                                                                                                                                           |
+| TIMESTAMP<br />TIMESTAMP_WITH_OUT_TIME_ZONE<br />TIMESTAMPTZ                                                      | LOCALDATETIME                                                                                                                                       |
+| GEOMETRY                                                                                                          | Not supported yet                                                                                                                                   |
+
+## Options
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/redshift.md:50]: Source Options

File: docs/en/connector-v2/source/redshift.md [hunk 26]
```diff
@@ -0,0 +1,147 @@
...
+| DATE                                                                                                              | LOCALDATE                                                                                                                                           |
+| TIME<br />TIME_WITH_TIME_ZONE<br />TIMETZ                                                                         | LOCALTIME                                                                                                                                           |
+| TIMESTAMP<br />TIMESTAMP_WITH_OUT_TIME_ZONE<br />TIMESTAMPTZ                                                      | LOCALDATETIME                                                                                                                                       |
+| GEOMETRY                                                                                                          | Not supported yet                                                                                                                                   |
+
+## Options
```
Comment (zhilinli123) [inline docs/en/connector-v2/source/redshift.md:54]: Source Options

File: docs/en/connector-v2/source/redshift.md [hunk 27]
```diff
@@ -0,0 +1,147 @@
...
+
+|             name             |  type  | required |     default     |                                                                                                                            description                                                                                                                            |
```
Comment (zhilinli123) [inline docs/en/connector-v2/source/redshift.md:56]: ditto

**Comment by EricJoy2048** [comment]
@zhilinli123  Please fix ci error.

**Comment by zhilinli123** [comment]
> @zhilinli123 Please fix ci error.

To replace the current pr links to https://github.com/apache/seatunnel/pull/5498

**Comment by zhilinli123** [comment]
https://github.com/apache/seatunnel/pull/4721

**Pull request closed without merging**